src/main/java/org/jitsi/videobridge/transform/RtxTransformer.java

```diff
@@ -211,18 +224,42 @@ private int getNextRtxSequenceNumber(long ssrc)
      * Tries to find an SSRC paired with {@code ssrc} in an FID group in one
      * of the channels from {@link #channel}'s {@code Content}. Returns -1 on
      * failure.
-     * @param ssrc the SSRC for which to find a paired SSRC.
+     * @param pkt the {@code RawPacket} that holds the RTP packet for
+     * which to find a paired SSRC.
      * @return An SSRC paired with {@code ssrc} in an FID group, or -1.
      */
-    private long getPairedSsrc(long ssrc)
+    private long getPairedSsrc(RawPacket pkt)
```

Comment:
Since this is now properly implemented, can we rename it to something like getRtxSsrc?